src/test/java/com/box/sdk/BoxFolderTest.java

```diff
@@ -670,6 +670,54 @@ public void testGetFolderInfoSucceeds() throws IOException {
         Assert.assertEquals(modifiedByName, info.getModifiedBy().getName());
     }
 
+    @Test
+    @Category(UnitTest.class)
+    public void testGetRestrictedCollaborationFieldSucceeds() throws IOException {
+        String result = "";
+        final String folderID = "12345";
+        final String folderInfoURL = "/folders/" + folderID;
+
+        result = TestConfig.getFixture("BoxFolder/GetFolderInfoForCollaborationRestriction200");
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.get(WireMock.urlPathEqualTo(folderInfoURL))
+                .willReturn(WireMock.aResponse()
+                        .withHeader("Content-Type", "application/json")
+                        .withBody(result)));
+
+        BoxFolder folder = new BoxFolder(this.api, folderID);
+        BoxFolder.Info info = folder.getInfo();
+
+        Assert.assertTrue(info.getIsCollaborationRestrictedToEnterprise());
+    }
+
+    @Test
+    @Category(UnitTest.class)
+    public void testSetRestrictedCollaborationFieldSucceeds() throws IOException {
+        String result = "";
+        final String folderID = "12345";
+        final String folderInfoURL = "/folders/" + folderID;
+        JsonObject jsonObject = new JsonObject()
+                .add("is_collaboration_restricted_to_enterprise", true);
+
+        result = TestConfig.getFixture("BoxFolder/GetFolderInfoForCollaborationRestriction200");
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.get(WireMock.urlPathEqualTo(folderInfoURL))
+                .willReturn(WireMock.aResponse()
+                        .withHeader("Content-Type", "application/json")
+                        .withBody(result)));
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.put(WireMock.urlPathEqualTo(folderInfoURL))
```

Comment:
It doesn't look like this is actually used in this test — can it be removed?  Could we move it to a test that actually asserts about the PUT behavior?